iron-validatable-behavior.html

```diff
@@ -91,7 +91,11 @@
      * @return {boolean} True if `values` is valid.
      */
     validate: function(values) {
-      var valid = this._validator && this._validator.validate(values);
+      var valid = true;
+      if (this.hasValidator()) {
+        valid = this._validator && this._validator.validate(values);
```

Comment:
You don't need to check for `this._validator` again after checking `this.hasValidator()`.